ee/util.go

```diff
@@ -10,12 +10,14 @@ var isEnterprise bool
 
 // IsEnterprise - checks if enterprise binary or not
 func IsEnterprise() bool {
```

Comment:
this function is actually unused now (after latest refactor) so you can just remove it as well.